jerry-core/ecma/builtin-objects/ecma-builtin-helpers.h

```diff
@@ -262,7 +262,7 @@ typedef ecma_value_t (*ecma_builtin_helper_sort_compare_fn_t) (ecma_value_t lhs,
                                                                ecma_value_t rhs, /**< right value */
                                                                ecma_value_t compare_func, /**< compare function */
                                                                ecma_object_t *array_buffer_p /**< arrayBuffer */
-                                                               );
+);
```

Comment:
Should be
`ecma_object_t *array_buffer_p); /**< arrayBuffer */`

Reply:
It cannot be ` ecma_object_t *array_buffer_p); /**< arrayBuffer */` bacause doxygen will complain that `array_buffer_p` is not documented, because the comment is outside the parentheses. However `ecma_object_t *array_buffer_p /**< arrayBuffer */);` is will work.